client/post-editor/test/post-editor.jsx

```diff
@@ -56,6 +56,10 @@ mockery.registerMock( 'my-sites/drafts/draft-list', MOCK_COMPONENT );
 mockery.registerMock( 'lib/layout-focus', {
 	set() {}
 } );
+// TODO: REDUX - add proper tests when whole post-editor is reduxified
```

Comment:
We can add tests where we spy that the expected redux function has been called in certain editor actions. There's no need to wait for the action to be fully implemented. If there isn't much scaffolding for editor unit tests, this may be a bit of work.